test/suites/external_auth.sh

```diff
@@ -31,6 +32,6 @@ EOF
 
     # cleanup
     lxc config unset candid.api.url
-    lxc config unset core.https_address
```

Comment:
why did this line go? not needed?

Reply:
Hmmm well it's not needed. I found that the RBAC test was failing because the server wasn't listening on `LXD_ADDR`, but that should be handled in the RBAC test itself with an `ensure_has_localhost_remote`. I will re-add it.

Reply:
Using `ensure_has_localhost_remote` makes sense

Reply:
Done.